.env.example

```
@@ -2,5 +2,6 @@ DATABASE_URL="sqlite:schwab.db"
 WS_RPC_URL=wss://arbitrum-one-rpc.publicnode.com
 ORDERBOOK=0xf6fdcb30fd027c73cbacd54e22f02987e3116ebc
 ORDER_HASH=0xf1696030ef3f39647663ba95d8e9a8a8d55da3e008a7979e8aae34473fe91657
+DEPLOYMENT_BLOCK=353562229
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The DEPLOYMENT_BLOCK key should go before the ORDERBOOK key

(UnorderedKey)